### ISSUE TITLE
The version of tensorflow need to be updated in ReadMe

### ISSUE THREAD

**Issue by GuanxiongLiu** [devsisters/pointer-network-tensorflow#8]
In my test, this project need the api version 1.0 for tensorflow. The ReadMe file need to update this information for user's convenience.

Thanks

**Comment by zhouliang-yu** [comment]
Yes, but if you are using the latest or 2.0+ version, you can just replace 
`from tensorflow.contrib import layers` to 
`from tensorflow.keras import layers`
